src/css/Home.css

```diff
@@ -0,0 +1,84 @@
+/* Margin */
+.margin {
+  padding-left: 9em;
+  padding-top: 20em;
+}
```

Comment:
Please use a descriptive name for this css class (if this file is still being used), and although they both create space, remember that margin and padding are **not** the same

The difference can be found [here](https://blog.hubspot.com/website/css-margin-vs-padding).

Reply:
removed